client/index.jsx

```diff
@@ -10,27 +10,30 @@ class App extends React.Component {
     this.state = {
       value: null
     }
+    this.handleGameMouseEntry = this.handleGameMouseEntry.bind(this)
```

Comment:
I'd opt for arrow functions here instead of using `.bind()` as I find them much more readable and they were specifically introduced to help you avoid using `.bind()`. This is much more of a minor quibble though and kind of a stylistic choice, so feel free to disregard if you feel strongly that this should remain as-is.